security/security-core/src/main/java/io/camunda/security/configuration/OidcAuthenticationConfiguration.java

```diff
@@ -22,6 +22,7 @@ public class OidcAuthenticationConfiguration {
   private String authorizationUri;
   private Set<String> audiences;
   private String usernameClaim = "sub";
+  private String applicationIdClaim;
```

Comment:
:wrench: We should probably validate that it's not the same claim as `usernameClaim`